calibrators/tag_based_pnp_calibrator/README.md

```diff
@@ -0,0 +1,123 @@
+| `max_tag_distance`                            | `double`              | `20.0`                           | Maximum allowed distance in meters from the camera to the tags.                                                                                                                                                                                                         |
+| `max_allowed_homography_error`                | `double`              | `0.5`                            | apriltag detection is discarded if the homography error is larger than `max_allowed_homography_error`.                                                                                                                                                                  |
+| `use_receive_time`                            | `bool`                | `false`                          | Flag to determine whether to use the receive time instead of the header timestamps.                                                                                                                                                                                     |
+| `use_rectified_image`                         | `bool`                | `true`                           | Flag to determine whether the input images are treated as rectified or not.                                                                                                                                                                                             |
+| `calibration_crossvalidation_training_ratio`  | `double`              | `0.7`                            | The ratio of data used for training versus validation during the calibration's cross-validation process.                                                                                                                                                                |
+| `calibration_convergence_min_pairs`           | `int`                 | `9`                              | The minimum number of apriltag and lidartag detection pairs required to consider the calibration process as potentially converged.                                                                                                                                      |
+| `calibration_convergence_min_area_percentage` | `double`              | `0.005`                          | Minimum percentage of the area that needs to be covered by detection.                                                                                                                                                                                                   |
```

Comment:
detectionS

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/61b823f4ed185b0b02665d17e4cca866a7bd9971